samples/sensor/hts221/sample.yaml

```diff
@@ -1,14 +1,23 @@
 sample:
   name: HTS221 Temperature and Humidity Monitor
-tests:
-  sample.sensor.hts221:
+common:
     harness: console
+    filter: dt_compat_enabled("st,hts221")
     tags: sensors
     depends_on: i2c hts221
     harness_config:
         type: multi_line
         ordered: yes
         regex:
-            - "Temperature: (.*)"
-            - "Relative Humidity: (.*)"
-        fixture: fixture_i2c_hts221
+            - "Observation:(.*)"
+            - "Temperature:(.*) C"
+            - "Relative Humidity:(.*)%"
+tests:
+  sample.sensor.hts221:
+    extra_configs:
+        - CONFIG_HTS221_TRIGGER_NONE=y
+  sample.sensor.hts221.trigger:
+    integration_platforms:
+        - disco_l475_iot1
```

Comment:
Move this to common so both tests have it as an integration platform?

Reply:
Right, done